www/include/monitoring/downtime/listDowntime.php

```diff
@@ -179,18 +177,25 @@
         $centreon->user->access->getAccessGroupsString() . ") ";
 }
 $request .= (isset($search_service) && $search_service != "" ? "AND 1 = 0 " : "") .
-    (isset($host_name) && $host_name != "" ? "AND h.name LIKE '%$host_name%' " : "") .
-    (isset($search_output) && $search_output != "" ? "AND d.comment_data LIKE '%$search_output%' " : "") .
+    "AND h.name LIKE ? " .
+    "AND d.comment_data LIKE ? " .
     (isset($view_all) && $view_all == 0 ? "AND d.end_time > '" . time() . "' " : "") .
     (isset($view_downtime_cycle) && $view_downtime_cycle == 0 ?
         " AND d.comment_data NOT LIKE '%Downtime cycle%' " : "") .
-    (isset($search_author) && $search_author != "" ? " AND d.author LIKE '%$search_author%'" : "") .
+    " AND d.author LIKE ?" .
     ") ORDER BY scheduled_start_time DESC " .
     "LIMIT " . $num * $limit . ", " . $limit;
```

Comment:
Are you sure that $num and $limit are sanitized ?

Reply:
it's not, I will manage it

Reply:
done